test/libsolidity/util/TestFileParser.h

```diff
@@ -111,9 +113,9 @@ class TestFileParser
 		char peek() const noexcept;
 
 		/// Returns true if the end of a line is reached, false otherwise.
-		bool isEndOfLine() const { return m_char == m_line.end(); }
+		bool eof() const { return m_char == m_source.end(); }
```

Comment:
Renamed this as it is misleading. It is not about end of line, but end of source.